core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/autoscales/AutoscaleV4Endpoint.java

```diff
@@ -74,6 +75,13 @@
             notes = Notes.STACK_NOTES, nickname = "getAutoscaleClusterByName")
     AutoscaleStackV4Response getAutoscaleClusterByName(@PathParam("name") String name);
 
+    @GET
+    @Path("/autoscale_cluster/name/{name}/internal")
+    @Produces(APPLICATION_JSON)
+    @ApiOperation(value = StackOpDescription.GET_INTERNAL_AUTOSCALE_BY_NAME, produces = APPLICATION_JSON,
+            notes = Notes.STACK_NOTES, nickname = "getInternalAutoscaleClusterByName")
+    AutoscaleStackV4Response getInternalAutoscaleClusterByName(@PathParam("name") String name, @AccountId @QueryParam("accountId") String accountId);
+
```

Comment:
This still needs to be interchanged.

Reply:
i changed the order of parameters in underlying methods

Reply:
sorry, right, will check

Reply:
should be ok now, stackname is the first parameter everywhere